hammer/utils/lib_utils.py

```diff
@@ -53,4 +53,6 @@ def get_headers(source: str) -> List[str]:
             # TODO: handle other compressed file types? Tools only seem to support gzip.
             fd = os.open(source, os.O_RDONLY)
             lines = os.pread(fd, nbytes, 0).splitlines()
-        return list(map(lambda l: l.decode('ascii', errors='ignore'), lines))
+        lines = list(map(lambda l: l.decode('ascii', errors='ignore'), lines))
+        all_lines = '\n'.join(lines).replace('\\\n','')  # combine lines that are split by \ character
```

Comment:
@harrisonliew This might be too hacky, but basically in one of the sky130 lib files (`sky130_ef_io__gpiov2_pad_wrapped_ss_ss_100C_1v60_3v00.lib`) the `capacitive_load_unit` declaration spanned 2 lines and it messed up the parsing in `LIBUtils.get_cap_unit()`
```
	capacitive_load_unit(1.000000, \
	  "pf");
```

Reply:
I think that's valid. But maybe the better approach is to remove line continuations (\) before processing lines. Can you also fix the type checker error?